types/index.ts

```diff
@@ -57,3 +57,8 @@ export interface INotes {
   timestamp: number;
   created_at: string;
 }
+
+export interface ISliceState {
+  notesArr: INotes[];
+  status: string;
```

Comment:
Good opportunity to use enum as type for status

type Status = "IDLE"  | "COMPLETE"

it will give you great autocompletetion

Reply:
Yes! I didn't notice this opportunity. Thanks.